get.sh

```diff
@@ -235,6 +235,9 @@ getTestKitGenAndFunctionalTestMaterial()
 	mv openj9/test/Utils Utils
 	mv openj9/test/functional functional
 	rm -rf openj9
+    if [ -d functional-AdoptOpenJDK ]; then
+        mv functional-AdoptOpenJDK/* functional
+    fi
```

Comment:
To have a root dir `functional-AdoptOpenJDK` is very confusing. It does not go with our overall structure. Please keep it `functional`.
We just need two functional folders to overwrite each other.

Reply:
* Rename from functional-AdoptOpenJDK to functional 
* get.sh was edited.
See https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/536/console